app/controllers/users.js

```diff
@@ -32,6 +32,13 @@ exports.getUsers = (req, res, next) => {
     .catch(error => next(error));
 };
 
+exports.invalidateAll = (req, res, next) => {
+  usersService
+    .invalidateAll(jwt.validate(req.headers.authorization))
```

Comment:
Why do you need to validate again the token ?

Reply:
It's actually a decode, the name just stuck from older functionalities but it doesn't fit anymore. I'll change it

Reply:
You must decode and validate in the middleware, not in each controller